application/backend/src/main/resources/libs/bulma/components/navbar/navbar.html

```diff
@@ -17,7 +17,7 @@
 */-->
 
 <sly data-sly-use.model="pl.ds.bulma.components.models.navbar.NavbarComponent">
-  <sly data-sly-set.classes="navbar ${model.navbarStyle}"></sly>
+  <sly data-sly-set.classes="navbar ${model.navbarStyle} ${model.containerStyle}"></sly>
```

Comment:
We need an additional <div class="container ${model.containerStyle}"> below <nav>